### PR TITLE
[feat] [doc] PIP-364: Introduce a new load balance algorithm AvgShedder

### PR DESCRIPTION
**Pull request by thetumbled**
### ✅ Contribution Checklist

- [x] I read the [contribution guide](https://pulsar.apache.org/contribute/document-contribution/)
- [x] I updated the [versioned docs](https://pulsar.apache.org/contribute/document-contribution/#update-versioned-docs)

### CODE REVIEW

File: docs/concepts-broker-load-balancing-concepts.md [hunk 1]
```diff
@@ -492,40 +492,41 @@ Unload all bundles in a namespace
...
+TransferShedder|Unload bundles from the **highest** load brokers to the **lowest** load brokers until the standard deviation of the broker load distribution is below the configured threshold.| This is the **default** strategy for the **extensible** load balancer. <br/><br/>It pre-assigns destination brokers when unloading. |Pulsar 3.0 and later versions.<br/><br/>This strategy is **only available** in the **extensible** load balancer.
+AvgShedder|Unload bundles to keep the range of broker resource usage within the configured threshold.| Use when you want to achieve greate service stability and load switching accuracy.                                                  |Pulsar 3.0 and later versions. <br/><br/>If you run version greater than 2.9 but below 3.0, you can try to cherry pick into your repository easily: https://github.com/apache/pulsar/pull/22946<br/><br/>This strategy is only available in the modular load balancer.
```
Comment (Demogorgon314) [inline docs/concepts-broker-load-balancing-concepts.md:505]: Can you describe which specific version supports the `AvgShedder`.

File: versioned_docs/version-3.2.x/concepts-broker-load-balancing-concepts.md [hunk 2]
```diff
@@ -552,12 +553,61 @@ Pulsar introduced TransferShedder to utilize the bundle transfer protocol from t
...
+When the difference in scores between two paired brokers exceeds the `loadBalancerAvgShedderLowThreshold` by
+`loadBalancerAvgShedderHitCountLowThreshold` times, or exceeds the `loadBalancerAvgShedderHighThreshold` by
+`loadBalancerAvgShedderHitCountHighThreshold` times, a bundle unload is triggered.
+For example, with the default value, if the score difference exceeds 15, it needs to be triggered 8 times continuously,
+and if the score difference exceeds 40, it needs to be triggered 2 times continuously.
+
+The larger the load difference between brokers, the smaller the number of times it takes to trigger bundle unloads,
+which can adapt to scenarios such as cluster rolling restart or expansion.
+
+For implementation details, see  [PIP-364: Introduce a new load balance algorithm AvgShedder](https://github.com/apache/pulsar/pull/22949).
```
Comment (Demogorgon314) [inline versioned_docs/version-3.2.x/concepts-broker-load-balancing-concepts.md:602]:
```suggestion
For implementation details, see [PIP-364: Introduce a new load balance algorithm AvgShedder](https://github.com/apache/pulsar/pull/22949).
```

**Review by heesung-sn — APPROVED** [review]
LGTM.

I think we better separate the format change from this PR.

**Comment by thetumbled** [comment]
> LGTM.
> 
> I think we better separate the format change from this PR.

I try serveral times to avoid the format change, but it seems that IDEA will format forcefully and automatically, i can't find out the switch to turn it off. Does anyone have any idea on solving this?

File: docs/concepts-broker-load-balancing-concepts.md (hunk 1 above)
Comment (thetumbled) [reply docs/concepts-broker-load-balancing-concepts.md]: Updated.

**Review by Demogorgon314 — APPROVED** [review]

**Pull request merged**